swatch-billable-usage/src/main/java/com/redhat/swatch/billable/usage/admin/api/InternalBillableUsageResource.java

```diff
@@ -51,6 +51,7 @@ public class InternalBillableUsageResource implements DefaultApi {
   private final InternalBillableUsageController billingController;
   private final EnabledOrgsProducer enabledOrgsProducer;
   private final ApplicationConfiguration configuration;
+  private final InternalBillableUsageController internalBillableUsageController;
```

Comment:
Why not just use existing `private final InternalBillableUsageController billingController;`? This can create circular dependency in future.